spec/data_utils_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'spec_helper'
+require_relative '../dependency'
+
+RSpec.describe DataUtils do
+  include described_class
```

Comment:
Please test module via `DummyClass` https://stackoverflow.com/questions/1542945/testing-modules-in-rspec